components/omega/src/infra/logging.h

```diff
@@ -0,0 +1,108 @@
+// Omega logging header file.
+
+#ifndef OMEGA_LOG_H
+#define OMEGA_LOG_H
+
+#pragma once
```

Comment:
with the header guards, what is the function of this pragma?

Reply:
You're right. It is just duplication of the same function. I will remove it.